packages/next-swc/crates/core/src/lib.rs

```diff
@@ -130,6 +134,10 @@ pub fn custom_before_pass(
             page_config::page_config(opts.is_development, opts.is_page_file),
             !opts.disable_page_config
         ),
+        match &opts.relay {
```

Comment:
Please make this non-wasm only

Reply:
@kdy1 How do I ensure this isn't applied in the wasm build? Is it like a `#[cfg(napi)]` thing?

Reply:
Yes, I mean `#[cfg(???)]`